css-viewport/Overview.bs

```diff
@@ -356,6 +360,151 @@ only the value previously set to it.
 }
 </pre>
 
+<h2 id='zoom-property'>
+	The 'zoom' property
+</h2>
+
+An element becomes zoomed when the 'zoom' property has a positive computed value different than 1.
+'zoom' affects computed property values and inherited property values.
+The computed value of 'zoom' is applied as a scalar to the used value of <<length>> CSS properties for the element and its descendants
+(including inherited properties such as 'line-height' or 'font-size'),
+plus intrinsic sizing of all replaced elements,
+background images,
+and nested frames (except for fenced frames [[!FENCED-FRAME]]),
+resulting in a magnification or minification effect.
+
+Nested values of 'zoom' multiply, resulting in additional scaling of <<length>> values.
```

Comment:
Ok I guess you kinda fix it up here, but that's unfortunate.

Reply:
I added a parenthetical above, does that resolve the concern? Happy to rephrase to be more clear if you have a suggestion..

Reply:
Oh you're referring to the computed value vs effective zoom...

Reply:
I added a sentence about the used value, LMK if that helps.